core/server/common/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -54,20 +55,25 @@ public boolean validate(Map<String, String> optionsMap) {
       return true;
     }
     if (!validateHdfsSettingParity(optionsMap)) {
-      System.err.format("Hdfs setting do not match.");
+      System.err.format("Hdfs setting do not match.%n");
       return false;
     }
     return true;
   }
 
   /**
-   * Checks whether this validation task should be skipped.
-   *
-   * @return true if underFS is NOT HDFS, false otherwise
+   * {@inheritDoc}
    */
-  protected boolean shouldSkip() {
-    String underFSURI = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS).toLowerCase();
-    return !underFSURI.startsWith("hdfs://");
+  @Override
+  public boolean shouldSkip() {
+    String scheme =
+        new AlluxioURI(Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS)).getScheme();
+    if (scheme == null || !scheme.startsWith("hdfs")) {
+      System.err.format("Root underFS is %s, not HDFS. Skipping validation for HDFS properties.%n",
```

Comment:
this could print something like "Root underFS is null, not HDFS. Skipping validation for HDFS properties.". However, it could just be that the underFS is local filesystem, which doesn't have a scheme

Reply:
Done.